blender/arm/material/make_transcluc.py

```diff
@@ -0,0 +1,51 @@
+import bpy
```

Comment:
Similar problem as with `Shaders/deferred_light.frag.glsl`, the filename contains a typo and thus git adds a completely new and unused file.